### PR TITLE
Changed flag name to be more descriptive added example to ReadMe.

### PR DESCRIPTION
**Pull request by guillermo-menjivar**

### CODE REVIEW

**Comment by gsauthof** [comment]
Thanks for the pull-request.

I see how the option name `--no-path` is not necessarily very intuitive. But the proposed `--disable-default-path` seems too much to type.

Also I would prefer to name `--no-path` and `--no-wlist` in the same style.

What do you think about introducing

```
--clear-path/--clear-wlist
```

or

```
--reset-path/--reset-wlist
```

instead?

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ Display the dependencies of a PE library:
     libwinpthread-1.dll
     USER32.dll
 
+Display the dependencies of a PE library by pointing to a custom path and excluding default path:
+  
+    Note: peldd uses /usr/x86_64-w64-mingw32/sys-root/mingw/bin/ as its default path.
```
Comment (gsauthof) [inline README.md:37]: The previous line shouldn't be part of the command block.

**Comment by guillermo-menjivar** [comment]
Yea I like "--clear-*" this provides enough info. And yea too much typing is not good for you, specifically midnight coding :) . Thanks for getting back so soon. Let me know if you want me to update my PR to what you decide and the correct info in the readme.

**Comment by gsauthof** [comment]
If it isn't too much hassle please go ahead and update (i.e. rebase) this pull-request such that `--clear-` is used for both options and in the readme. I'll merge it then.

**Comment by guillermo-menjivar** [comment]
👍

**Comment by guillermo-menjivar** [comment]
you will see update sometime tonight

**Comment by gsauthof** [comment]
@videte47 - afaict this pull request wasn't updated - can you check and (force) push again?

**Comment by gsauthof** [comment]
Closed this by accident.

**Pull request closed without merging**